services/cdnjs/cdnjs.js

```diff
@@ -43,7 +50,8 @@ module.exports = class Cdnjs extends BaseJsonService {
   static get examples() {
     return [
       {
-        previewUrl: 'jquery',
+        staticExample: this.formatBadge('1.5.2'),
+        exampleUrl: 'jquery',
```

Comment:
If the `render` abstraction is pushed into the base class, we could put the properties here.

```js
return [{
  props: { version: '1.5.2' }, // formatBadge gets called automatically
  exampleUrl: 'jquery',
  ...
}];
```

Replacing the function call would make this easier to use with static properties when they come along.